src/components/UpcomingTalks/index.js

```diff
@@ -0,0 +1,79 @@
+import React from "react"
+import { graphql, useStaticQuery } from "gatsby"
+
+import Card from "../Card"
+import downloadCalendarFile from "../../utils/downloadCalendarFile"
```

Comment:
[You could skip this]

Check whether you can create **neater imports** from common places such as images, icons or utils can modified via gatsby webpack config i.e.;

FROM "../../utils/downloadCalendarFile"  TO "utils/downloadCalendarFile"

Maybe include this as a separate ticket

Reply:
I agree, I think this needs to be a separate ticket